account-setup.rst

```diff
@@ -85,9 +85,11 @@ If a secondary user is not already listed, add one now with the *New user* butto
 Landscape
 ---------
 
-You will have received an email message inviting you to activate your Landscape account (Subject: "You have been invited to the Landscape account […]", from ``noreply+landscape@canonical.com``). 
+If you have a paid Ubuntu Pro subscription and would like to use Landscape SaaS, contact your Account Manager or Canonical’s Customer Success team to request an invitation to join your Landscape account.
```

Comment:
add cross linke to "Self-hosted landscape" in understanding landscape section to address self hosted landscape